portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACPagePath.macro

```diff
@@ -0,0 +1,54 @@
+definition {
+
+	@summary = "Clears segment search filter"
+	macro clearSegmentSearchFilter() {
+		Type.sendKeys(
+			locator1 = "ACSettings#SEARCH_BAR",
+			value1 = "keys=CONTROL,a");
+
+		KeyPress(
+			locator1 = "ACSettings#SEARCH_BAR",
+			value1 = "\BACK_SPACE");
+	}
+
+	@summary = "Selects the segment in page filter"
+	macro clickFilterSegment(segmentName = null) {
+		Click(
+			key_segmentName = ${segmentName},
+			locator1 = "ACUtils#PAGE_SEGMENT_FILTER");
+	}
+
+	@summary = "Removes the segment in page filter"
+	macro removeSegmentFilter() {
+		Click(locator1 = "ACSites#REMOVE_SEGMENT_FILTER");
+	}
+
+	@summary = "Asserts the total of page path"
+	macro viewPagePathTotal(pageTotal = null) {
+		var key_pageTotal = ${pageTotal};
+
+		AssertElementPresent(
+			locator1 = "ACSites#PAGE_PATH_TOTAL",
+			value1 = ${pageTotal});
+	}
+
+	@summary = "Asserts the dropdown segment filter"
+	macro viewSegmentDropdown(nameList = null) {
```

Comment:
Hey @joshitagaki, I think the `nameList` variable is always necessary to use this new macro, right? So I think we can set it without `null` here so it can be a required variable

```suggestion
	macro viewSegmentDropdown(nameList) {
```